samples/benchmarks/coremark/boards/Kconfig.sysbuild_boards

```diff
@@ -7,10 +7,35 @@
 if BOARD_NRF5340DK_NRF5340_CPUAPP
 
 config APP_CPUNET_RUN
-	bool "Run the CoreMark benchmark on the nRF5340 Network Core"
+	bool "Run the CoreMark benchmark on the nRF5340 network core"
 	default y
 
 config IMAGE_2_BOARD
 	default "nrf5340dk/nrf5340/cpunet" if APP_CPUNET_RUN
 
 endif
+
+if BOARD_NRF54H20DK_NRF54H20_CPUAPP
+
+config APP_CPUNET_RUN
+	bool "Run the CoreMark benchmark on the nRF54H20 radio core"
+	default y
+
+config APP_CPUPPR_RUN
+	bool "Run the CoreMark benchmark on the nRF54H20 PPR core"
+	depends on !APP_CPUNET_RUN
+	help
+	  Due to the limited number of UART ports, you can run the CoreMark benchmark only on two
+	  cores at a time; application core and either radio core or PPR core.
+
+	  PPR code is run from MRAM with the execution in place (XIP) method. You must use the
+	  "nordic-ppr-xip" snippet for the application core to be able to boot the PPR core.
+	  You need to provide the "coremark_SNIPPET=nordic-ppr-xip" build argument.
+
+config IMAGE_2_BOARD
+	default "nrf54h20dk/nrf54h20/cpurad" if APP_CPUNET_RUN
+
+config IMAGE_3_BOARD
+	default "nrf54h20dk/nrf54h20/cpuppr" if APP_CPUPPR_RUN
```

Comment:
this should not be a board target but a qualifier, so that it will adjust to any base board (e.g. customers), use cmake to select the correct board target see https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/netcore.cmake for an example

Reply:
Yes, we are aware of it and we have a task to improve it: https://nordicsemi.atlassian.net/browse/NCSDK-26813. This PR follows the build approach that has already been taken earlier.

Reply:
@tejlmand, I'm tagging you since Jamie is on vacation. Do you think we could merge it without applying the Jamie's suggestion? I'm really swamped with other tasks for the NCS 2.7.0 release. I'd love to align this after the release.

Reply:
@nordicjm, I cleaned up Kconfigs in the CoreMark sample on your request. Please take a look